src/debug_input.cpp

```diff
@@ -113,7 +113,8 @@ void DebugInput::debug_movement_keys() {
 }
 
 void DebugInput::debug_movement_mouse(events::WindowCursorData d) {
-
+	if (keys[GLFW_KEY_LEFT_CONTROL])
```

Comment:
i just needed a way to take a screenshot of the game with Windows Snipping Tool without my mouse movement causing the game's camera to move. lol.